docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+---
+title: ML Models in MindsDB
+---
+
+In this section, we go through the ML model types available in MindsDB. These are regression models, classification models, and time series models.
+
+## What is an ML Model?
+
+A machine learning (ML) model is a program trained using the available data in order to learn how to recognize patterns and behaviors and make decisions that lead to predicting future data.
+
+You input data into an ML model. It processes this input data, searching for patterns and correlations. After that, the ML model returns output data defined based on the input data.
+
+### Features
+
+Features define the columns or variables that an ML model uses as input data to search for patterns and predict the target variable.
+
+### Target
+
+A target is a column or variable that an ML model predicts based on the information fetched out of the features.
+
+### Training Dataset
+
+The training dataset is used during the training phase of an ML model. It contains both feature variables and a target variable. As its name indicates, it is used to train an ML model.
+
+The ML model takes the entire training dataset as input. It learns the patterns and relationships between feature variables and correlates them with the target variable values.
+
+Once the training process is complete, one can move on to the validation phase.
+
+### Validation Dataset
+
+The validation dataset is used during the validation phase of an ML model. It contains both feature variables and a target variable, like the training dataset. But as its name indicates, it is used to validate the predictions made by an ML model.
+
+The ML model takes only the feature variables from the validation dataset as input. Based on what the model learns during the training process, it makes predictions for the values of a target variable.
+
+Now comes the validation step. To assess the accuracy of the ML model, one compares the target variable values from the validation dataset with the target variable values predicted by the ML model. The closer these values are to each other, the better accuracy of the ML model.
+
+### Input Dataset
+
+After completing the training and validation phases, one can provide the input dataset consisting of only the feature variables to predict the target variable values.
+
+## How an ML Model is Created?
```

Comment:
Change to:

> ## How is an ML model created?

Reply:
Changed (but kept the capitalized letters to be consistent):

> ## How is an ML Model Created?